engines/bastion/app/views/bastion/layouts/assets.html.erb

```diff
@@ -10,11 +10,13 @@
 <% end %>
 
 <% content_for(:javascripts) do %>
-  <%= javascript_include_tag 'bastion/bastion' %>
+
+  <%=  javascript_tag("window.tfm.tools.loadPluginModule('/webpack/katello','katello','./common_index');".html_safe) %>
+  <%= javascript_include_tag('bastion/bastion' , {type: 'module'})%>
   <% if File.exist?("#{Bastion::Engine.root}/vendor/assets/javascripts/#{Bastion.localization_path(I18n.locale)}") %>
     <%= javascript_include_tag(Bastion.localization_path(I18n.locale)) %>
   <% end %>
-  <script type="text/javascript">
+  <script type="module">
```

Comment:
What defines this type? Is it webpack itself or something else?

Reply:
it looked more correct to use `module` than `text/javascript` https://developer.mozilla.org/en-US/docs/Web/JavaScript/Guide/Modules